charts/redpanda/ci/31-overwrite-statefulset-pod-labels.yaml

```diff
@@ -0,0 +1,3 @@
+statefulset:
+  podTemplateLabels:
+    azure.workload.identity/use: "true"
```

Comment:
suuuuuuuuuuuper nit: Please add a newline to the end of this file. If you open it in vim, save, and close it that'll do it.

Reply:
Np. I will add new line.